src/SharpCompress/Utility.cs

```diff
@@ -229,15 +229,18 @@ public static DateTime DosDateToDateTime(Int32 iTime)
             return DosDateToDateTime((UInt32)iTime);
         }
 
-        public static long TransferTo(this Stream source, Stream destination)
+        public static long TransferTo(this Stream source, Stream destination, Action<long, int> action = null)
```

Comment:
This Utility won't be reused and it's just for the library.  I'm fine with internal dependencies.